lib/stripe/subscriptions.ex

```diff
@@ -91,6 +91,19 @@ defmodule Stripe.Subscriptions do
 
   ## Example
   ```
+  Stripe.Subscriptions.change "customer_id", "subscription_id", [plan: "plan_id"]
+  ```
+  """
+  def change(customer_id, sub_id, opts) when is_list(opts) do
```

Comment:
I did consider that and the new tests with opts (with and without a key) pass for me, as well as the existing tests you have. That function head is on line 125.

Can you explain more what you mean about pattern matching on `opts`? Are you suggesting we allow only keys allowed by the Stripe API?